html/js/i18n.js

```diff
@@ -596,10 +599,11 @@ const i18n = {
       redisQueueSize: 'Redis Queue Size',
       refresh: 'Refresh',
       refreshAttachmentsHelp: 'Refresh to view all recently added attachments for this case.',
+      refreshCaseHistoryHelp: 'Refresh to view the latest history for this case.',
+      refreshDetectionHistoryHelp: 'Refresh to view the latest history for this case.',
```

Comment:
Should this say "for this detection" or "for this rule" rather than "case"?

Reply:
Yep, didn't follow through on my thought there.